app/controllers/application_controller.rb

```diff
@@ -7,7 +7,23 @@ class ApplicationController < ActionController::Base
   before_filter :authorize!
   before_filter :fedora_setup
 
-  rescue_from ActiveFedora::ObjectNotFoundError, with: -> { render text: 'Object Not Found', status: :not_found }
+  rescue_from ActionController::BadRequest do
+    msg = "Bad request: #{params}"
+    logger.error msg
+    render text: msg, status: :bad_request
+  end
+
+  rescue_from ActiveFedora::ObjectNotFoundError do
+    msg = "Object Not Found: #{params}"
+    logger.error msg
+    render text: msg, status: :not_found
+  end
+
+  rescue_from Rubydora::FedoraInvalidRequest do
+    msg = "Fedora Rejected Object Request: #{params}"
+    logger.error msg
+    render text: msg, status: :not_found
```

Comment:
this seems more like a 400 to me:
"The server cannot or will not process the request due to something that is perceived to be a client error (e.g., malformed request syntax, invalid request message framing, or deceptive request routing)."
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error
https://tools.ietf.org/html/rfc7231#section-6.5.1